### ISSUE TITLE
Unable to preview site link

### ISSUE THREAD

**Issue by yangszuyu** [jlyst/gallery#9]
---
name: Usability issue
about: Identify an issue that compromises the user experience.
title: Unable to preview site link
labels: usability issue
assignees: ''

---
When it comes to preview the site link, it turns into the grey block, it only shows the link but unable to preview the site.
 
<img width="815" alt="screen shot 2019-02-15 at 9 02 45 am" src="https://user-images.githubusercontent.com/43279908/52861692-48f11c00-3101-11e9-9469-9dace0d624f2.png">

**Comment by jlyst** [comment]
Since the preview uses an iFrame to preview the link, only certain sites will display correctly. For example, google dissallows it's content to be viewed in an iFrame. So the gallery is programmed to show the link by default--unless it is to repl.it, which does display in an iframe. 

Do you suggest we display a link to be different in the default case, rather than a large box?

**Comment by yangszuyu** [comment]
Right, I found that in some cases like the link I show below, it would be a bit weird for mentors to learn what the content really as below. 
<img width="396" alt="screen shot 2019-02-28 at 6 50 46 pm" src="https://user-images.githubusercontent.com/43279908/53606774-019b6e80-3b8a-11e9-8d8f-6b8fea385347.png">

What about showing the link directly below instead of placing in a large box?
